plugin/pkg/admission/imagepolicy/admission_test.go

```diff
@@ -1058,6 +1060,16 @@ func TestReturnedAnnotationAdd(t *testing.T) {
 			attr = &fakeAttributes{attr, annotations}
 
 			err = wh.Validate(context.TODO(), attr, nil)
+			if tt.wantErr {
+				if err == nil {
+					t.Errorf("%s: expected error making admission request: %v", tt.test, err)
+				}
+				return
```

Comment:
don't return here, it skips the annotations equality check which is the point of the test